detection_rules/etc/packages.yml

```diff
@@ -18,13 +18,13 @@ package:
     - size: 16x16
       src: /img/security-logo-color-64px.svg
       type: image/svg+xml
-    source:
-      license: Elastic-2.0
     name: security_detection_engine
     owner:
       github: elastic/protections
       type: elastic
+    source:
+      license: Elastic-2.0
     title: Prebuilt Security Detection Rules
     type: integration
-    version: 8.14.0-beta.0
+    version: 8.14.1-beta.1
```

Comment:
Why are these changes in here?

Reply:
Reverted them was generated when things were tested locally